pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -559,6 +559,10 @@ func (c *Compactor) runCompact(ctx context.Context, logger log.Logger, job Job,
 		return err
 	}
 
+	if !c.limits.BloomCompactorEnabled(job.tenantID) {
```

Comment:
I think this limit should be applied inside the loop over tenants in `compactUsers`. The limit should probably just print a warning (even info?) message and continue to the next tenant. This way you also skip doing a bunch of work for the tenant.

Reply:
Yeah i was torn in between, happy to move there.